src/server.js

```diff
@@ -3,34 +3,63 @@
 const express = require('express');
 const { logger } = require('./middleware/logger');
 
+require('./db');
+
 const { hello } = require('./handler/hello');
 const { data } = require('./handler/data');
-
-const makeError = (req, res) => {
-  throw new Error('This is the error handler!');
-};
-
-const pet = (req, res) => {
-  res.status(200).send({name: req.params.pet});
-};
+const { validator }= require('./middleware/validator');
+const {createBoxer, listBoxers, getBoxer, deleteBoxer, updateBoxer } = require('./routes/boxer');
+const { createCoder, listCoders, getCoder, deleteCoder, updateCoder } = require('./routes/coder');
+const { db } = require('./db');
 
 const app = express();
 
 app.use(logger);
+app.use(express.json());
 
 app.get('/', hello); //handler happens at the end and gives data back. Endpoint to your port (3000)
 app.get('/data', data); // '/data' is a handler which is a function itself. function takes 2 at once
-app.get('/throw-error', makeError);
-app.get('/pets/:pet', pet);
+// app.get('/throw-error', makeError);
+// app.get('/pets/:pet', pet);
+
+app.get('/boxer', listBoxers);
+app.post('/boxer', createBoxer);
+app.get('/boxer/:id', getBoxer);
+app.delete('/boxer/:id', deleteBoxer);
+app.put('/boxer/:id', updateBoxer);
+
+app.get('/coder', listCoders);
+app.post('/coder', createCoder);
+app.get('/coder/:id', getCoder);
+app.delete('/coder/:id', deleteCoder);
+app.put('/coder/:id', updateCoder);
```

Comment:
Move to routes files and replace with `app.use(boxerRoute)` and `app.use(coderRoute)` when hit it will go to your route files and do the CURDS